Submissions/Laguerta_Week3.playground/Contents.swift

```diff
@@ -0,0 +1,217 @@
+//: ## Functions and Closures
+//: 1.)
+//: Remove the `day` parameter. Add a parameter to include today’s lunch special in the greeting.
+//:
+//:  Write a custom argument label before the parameter name, or write `_` to use no argument label.
+//:
+func greet(_ person: String, _ lunch: String) -> String {
+    return "Hello \(person), today's lunch special is \(lunch)."
+}
+greet("John", "Red Curry")
```

Comment:
as concise as it is to not have an argument label, consider how readable your function is at the call site. Who is John Red Curry? What is John Red Curry? Maybe an argument label would help us understand the context. :D